cni/pkg/ebpf/server/redirectionServer.go

```diff
@@ -14,6 +14,8 @@
 
 package server
 
+// +build linux
```

Comment:
why the build tag vs `_linux.go` suffix on the name?

Reply:
Seemed less invasive, I have no strong preference for one or the other, if we like file suffixes I will do that.